### DISCUSSION TITLE
No SessionDestroyedMessage with the Tomcat RedissonSessionManager?

### DISCUSSION THREAD

**Discussion by ehardy** [redisson/redisson#5407] [Q&A] [answered]
Hello,

I'm testing the Tomcat `RedissonSessionManager` with multiple instances of a test application. I have put breakpoints in the different areas where `SessionCreatedMessage` and `SessionDestroyedMessage` messages are handled. Whenever I create a session on one Tomcat instance, the breakpoint for the `SessionCreatedMessage` message is hit on the second instance and the session is available from it. However, if I invalidate the session from the second Tomcat instance, where the session was not initially created on, there are no `SessionDestroyedMessage` message that get's published, none of the breakpoints get hit. However, if I destroy the session from the first instance, where the session was created, then there is a `SessionDestroyedMessage` message that gets published.

Is that the expected behaviour?

Thanks

**Comment by mrniko** [comment, marked as answer]
Fixed in https://github.com/redisson/redisson/issues/5408

**Reply by ehardy** [reply]
Awesome! Tried the latest master branch with the fix, all good. Thank you so much for the quick response.

**Reply by rahil-pandya** [reply]
Hi @mrniko, appreciate the quick fix! Any idea when this will be rolled out?

**Reply by ehardy** [reply]
Hello @mrniko , any idea when a release including this fix will be available? Thanks
